service_test_case.py

```diff
@@ -16,9 +16,12 @@
 from nio.util.discovery import is_class_discoverable as _is_class_discoverable
 from nio.util.runner import RunnerStatus
 from niocore.core.loader.discover import Discover
+
 from .router import ServiceTestRouter
 from .module_persistence_file.module import FilePersistenceModule
 from .module_persistence_file.persistence import Persistence
+from .scheduler.module import TestingSchedulerModule
```

Comment:
To stick with the naming convention of `module_persistence_file`, it makes sense to move this from `scheduler` into `module_scheduler_synchronous` or something like that.

Maybe change `TestingSchedulerModule` to something more descriptive as well, to show that what makes it special is that it's synchronous.

Maybe both of these `module_*` directories should move into a `module` directory?